src/interfaces/options/array.options.ts

```diff
@@ -4,4 +4,6 @@ export interface ArrayOptions {
     type: Constructor,
     maxItems?: number,
     minItems?: number,
+    nullable?: boolean,
```

Comment:
I chose not to use inheritance here, but we obviously could.

(We have tests that validate that every supported decorator recipe takes these two options.)